Essentials/src/com/earth2me/essentials/messaging/SimpleMessageRecipient.java

```diff
@@ -117,13 +117,14 @@ public MessageResponse onReceiveMessage(IMessageRecipient sender, String message
         if (!isReachable()) {
             return MessageResponse.UNREACHABLE;
         }
-        
+
         User user = getUser(this);
         boolean afk = false;
         if (user != null) {
-            if (user.isIgnoreMsg()
-                && !(sender instanceof Console)) { // Console must never be ignored.
+            if (user.isIgnoreMsg() && !(sender instanceof Console))  { // Console must never be ignored.
+              if (sender instanceof IUser && !((IUser) sender).isAuthorized("essentials.msgtoggle.bypass")) { // Don't ignore senders with permission
```

Comment:
I don't think it's possible for the sender to simultaneously be a `Console` and an `IUser` at the same time, so the previous `!(sender instanceof Console)` check is redundant.